mypy/newsemanal/semanal.py

```diff
@@ -3881,27 +3797,42 @@ def lookup_type_node(self, expr: Expression) -> Optional[SymbolTableNode]:
             return n
         return None
 
-    def get_module_symbol(self, node: MypyFile, parts: List[str]) -> Optional[SymbolTableNode]:
-        """Look up a symbol from the module symbol table."""
-        # TODO: Use this logic in more places?
+    def get_module_symbol(self, node: MypyFile, name: str) -> Optional[SymbolTableNode]:
+        """Look up a symbol from a module.
+
+        Return None if no matching symbol could be bound.
+        """
         module = node.fullname()
         names = node.names
         # Rebind potential references to old version of current module in
         # fine-grained incremental mode.
         if module == self.cur_mod_id:
             names = self.globals
```

Comment:
Since we are here again, could you please add a test that would fail without these two lines. Currently all tests pass if I remove them.

Reply:
I couldn't come up with a test that would require this. It's possible that fine-grained dependencies will deal with all the cases that this was supposed to handle. I've removed this now.